pkg/datastore/mongodb/mongodb.go

```diff
@@ -27,17 +29,23 @@ import (
 	"github.com/pipe-cd/pipe/pkg/datastore"
 )
 
-type MongoDB struct {
-	ctx      context.Context
-	client   *mongo.Client
-	database string
+const (
+	// Scram type ref https://github.com/mongodb/mongo-go-driver/blob/9e2aca8afd8821e6b068cc2f25192bc640d90a0d/mongo/client_examples_test.go#L119
+	// Default
+	Scram string = "SCRAM"
```

Comment:
nit: I feel like we can keep it private until needed.

Reply:
@nakabonne Thanks for your review, I change it to unexported one. Please check again when you have time :+1:  :+1: